DistributedTransaction/DistributedTransaction.tla

```diff
@@ -100,9 +100,15 @@ ReqMessages ==
   \union  [start_ts : Ts, primary : KEY, type : {"prewrite_optimistic"}, key : KEY]
   \union  [start_ts : Ts, primary : KEY, type : {"prewrite_pessimistic"}, key : KEY]
   \union  [start_ts : Ts, primary : KEY, type : {"commit"}, commit_ts : Ts]
-  \union  [start_ts : Ts, primary : KEY, type : {"cleanup"}]
   \union  [start_ts : Ts, primary : KEY, type : {"resolve_rollbacked"}]
   \union  [start_ts : Ts, primary : KEY, type : {"resolve_committed"}, commit_ts : Ts]
+  \* In TiKV, there's an extra flag "rollback_if_not_exist" in the check_txn_status
+  \* request.  If the primary key lock is missing (and no record in the write column),
+  \* there are two cases: the prewrite request of the primary key is delayed, or is lost
+  \* due to client (TiDB) crash.  To distinguish these two, it must wait until the
+  \* TTL of the lock to be resolved expired.  In the TLA+ spec, the TTL is considered 
+  \* constantly expired when the action is taken, so there's no need to model the flag.

```

Comment:
```suggestion
  \* In TiKV, there's an extra flag `rollback_if_not_exist` in the `check_txn_status` request.
  \*
  \* Because the client prewrites the primary key and secondary key in parallel, it's possible
  \* that the primary key lock is missing and also no commit or rollback record for the transaction
  \* is found in the write CF, while there is a lock on the secondary key (so other transaction
  \* is blocked, therefore this check_txn_status is sent). And there are two possible cases:
  \*
  \*    1. The prewrite request for the primary key has not reached yet.
  \*    2. The client is crashed after sending the prewrite request for the secondary key.
  \*
  \* In order to address the first case, the client sending `check_txn_status` should not rollback
  \* the primary key until the TTL on the secondary key is expired, and thus, `rollback_if_not_exist`
  \* should be set to false before the TTL expires (and true afterward).
  \*
  \* In TLA+ spec, the TTL is considered constantly expired when the action is taken, so the
  \* `rollback_if_not_exist` is assumed true thus no need to carry it in the message.
```